solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestRankSVMModel.java

```diff
@@ -34,7 +34,7 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class TestLTRScoringModel extends TestRerankBase {
+public class TestRankSVMModel extends TestRerankBase {
```

Comment:
moving this class from ltr.feature to ltr.model will follow separately for clarity